ral-facilities/scigateway
# Replace default favicon with STFC icon

## Description
Replaced the standard ReactApp favicon with the one used on STFC sites.

## Testing instructions
Launch the application and verify the correct icon is appearing in a browser tab.


## Agile board tracking
connect to #37
